src/hecs/mod.rs

```diff
@@ -3,5 +3,5 @@ pub mod frag_iter;
 pub mod heavy_compute;
 pub mod simple_insert;
 pub mod simple_iter;
-pub mod serialize_binary;
-pub mod serialize_text;
\ No newline at end of file
+// pub mod serialize_binary;
```

Comment:
why not just remove?

Reply:
Those changes were not intended to be in the PR at all.